cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/validator/CoreValidator.java

```diff
@@ -0,0 +1,311 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.template.etl.validator;
+
+import co.cask.cdap.api.annotation.Description;
+import co.cask.cdap.api.annotation.Name;
+import co.cask.cdap.api.annotation.Plugin;
+import co.cask.cdap.template.etl.api.Validator;
+import org.apache.commons.validator.GenericValidator;
+import org.apache.commons.validator.routines.CreditCardValidator;
+import org.apache.commons.validator.routines.DateValidator;
+import org.apache.commons.validator.routines.DomainValidator;
+import org.apache.commons.validator.routines.DoubleValidator;
+import org.apache.commons.validator.routines.EmailValidator;
+import org.apache.commons.validator.routines.FloatValidator;
+import org.apache.commons.validator.routines.ISBNValidator;
+import org.apache.commons.validator.routines.InetAddressValidator;
+import org.apache.commons.validator.routines.IntegerValidator;
+import org.apache.commons.validator.routines.LongValidator;
+import org.apache.commons.validator.routines.RegexValidator;
+import org.apache.commons.validator.routines.ShortValidator;
+import org.apache.commons.validator.routines.UrlValidator;
+
+/**
+ * Utility class containing validator functions provided by org.apache.commons.validator.routines package.
+ */
+@Plugin(type = "validator")
+@Name("core")
+@Description("Core Validator functions.")
+public class CoreValidator implements Validator {
+
+  private static final EmailValidator EMAIL_VALIDATOR = EmailValidator.getInstance();
+  private static final DateValidator DATE_VALIDATOR = new DateValidator();
+  private static final CreditCardValidator CREDIT_CARD_VALIDATOR = new CreditCardValidator();
+  private static final DoubleValidator DOUBLE_VALIDATOR = new DoubleValidator();
+  private static final IntegerValidator INTEGER_VALIDATOR = new IntegerValidator();
+  private static final FloatValidator FLOAT_VALIDATOR = new FloatValidator();
+  private static final ShortValidator SHORT_VALIDATOR = new ShortValidator();
+  private static final LongValidator LONG_VALIDATOR  = new LongValidator();
+  private static final UrlValidator URL_VALIDATOR  = new UrlValidator();
+  private static final DomainValidator DOMAIN_VALIDATOR = DomainValidator.getInstance();
+  private static final ISBNValidator ISBN_VALIDATOR = new ISBNValidator();
+  private static final InetAddressValidator INET_ADDRESS_VALIDATOR = new InetAddressValidator();
```

Comment:
i don't think we have that requirement to the variables be sorted in alphabetical order. also these are variables and quite self explanatory.